### ISSUE TITLE
model Buildings.Examples.Tutorial.SimpleHouse.SimpleHouse6 fails verification

### ISSUE THREAD

**Issue by AndreaBartolini** [OpenModelica/OpenModelica#12654]
The following variables of the model `Buildings.Examples.Tutorial.SimpleHouse.SimpleHouse6` fails the verification vs the Dymola reference file:
- `fan.sta_b.T`
- `fan.sta_a.T`
- `heaWat.u`
- 
**Temperatures**
![image](https://github.com/OpenModelica/OpenModelica/assets/25974985/5cec4cef-80e5-4cfa-a60b-1fb82542cbd5)
![image](https://github.com/OpenModelica/OpenModelica/assets/25974985/7cabf490-7b60-486f-998d-6d35b6c32d93)

herebelow the same temperatures obtained by running the simulation on the following local machine:
Win11 pro - 64 bit
Dymola Version 2024x, 2023-10-06
OpenModelica v1.24.0-dev-168-g907b8bd1e8 (64-bit)
Building master 0d8ed44f190fae7af6fa5e84001adc4b8ee2625d
![image](https://github.com/OpenModelica/OpenModelica/assets/25974985/f0b7cd3e-1961-4fa7-88fc-d3b7c8aecdf5)

The temperature `fan.sta_a.T` and `fan.sta_b.T` are calculated by the medium state by using the `actualStream()` operator, this becomes a problem in zero-flow condition where different compilers or compiler versions may produce different results.
In non-zero-flow condition the verification is satisfied.

Note that in this model the reverse flow is allowed, this is the reason because the  `actualStream()` operator is used instead of the `instream()` operator.

**heaWat.u**
![image](https://github.com/OpenModelica/OpenModelica/assets/25974985/271d5fb1-3b81-4949-8d8a-55c960673f79)

herebelow the same transient obtained by running the simulation on the above local machine:
![image](https://github.com/OpenModelica/OpenModelica/assets/25974985/fb1e124d-8983-4212-a527-2ffddb026324)

The problem in the CI seems to be related to the output interval, as in the issue #12653

**Comment by casella** [comment]
@mwetter if some variables cannot be uniquely determined, we should probably remove them from the reference results. Do you have other ideas?

**Comment by mwetter** [comment]
@casella : Yes, we should exclude the `sta_?.T` variables from these reference results as their are undefined at zero flow rate.
@hcasperfu: Can you please make this change in `IBPSA` and propagate it down to `Buildings`.

**Comment by casella** [comment]
I guess this will improve the verification results massively 😄

**Comment by AndreaBartolini** [comment]
The PR https://github.com/ibpsa/modelica-ibpsa/pull/1918 fixes the issue
